workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Ipc.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Tools.MiniJSON;
+
+namespace Improbable.Gdk.DeploymentManager
+{
+    public static class Ipc
+    {
+        public enum ErrorCode : uint
+        {
+            FileNotFound = 1,
+            Unauthenticated = 2,
+            NotFound = 3,
+            UnknownGrpcError = 4,
+            Other = 5
+        }
+
+        public class Error
+        {
+            public uint Code;
+            public string Message;
+
+            public static Error FromStderr(List<string> stderr)
+            {
+                if (stderr.Count == 0)
+                {
+                    throw new ArgumentException("Cannot parse error from empty stderr.");
+                }
+
+                // We expect only the first line to be valid.
+                var deserialized = Json.Deserialize(stderr[0]);
```

Comment:
This could fail, probably should be handled here rather than throwing

Reply:
Added an additional error code (`CannotParseOutput`) and catch exceptions that are thrown while parsing and propagate them upward.